ui/src/app/app.component.html

```diff
@@ -6,44 +6,38 @@
         <div *ngIf="currentUser != null && currentUser.roleID === 1">
             <a routerLink="/doctors" routerLinkActive="active">
                 <button mat-button>
-          <mat-icon>local_hospital</mat-icon>
-          Doctors
-        </button>
+                    <mat-icon>local_hospital</mat-icon>
+                    Doctors
+                </button>
             </a>
             <a routerLink="/patients" routerLinkActive="active">
                 <button mat-button>
-          <mat-icon>assignment_ind</mat-icon>
-          Patients
-        </button>
+                    <mat-icon>assignment_ind</mat-icon>
+                    Patients
+                </button>
             </a>
-            <a routerLink="/meds" routerLinkActive="active">
+            <a routerLink="/interviews/" routerLinkActive="active">
```

Comment:
```suggestion
            <a routerLink="/interviews" routerLinkActive="active">
```

Reply:
It doesn't really change anything in the behavior, but IMO the `/` sign is irrelevant and we should try to stick to the same coding style all over the project.